build/minikube/run-local.sh

```diff
@@ -10,4 +10,6 @@ echo "Using KUBECONFIG '$KUBECONFIG'"
 kubectl create ns ${NAMESPACE} || true
 kubectl apply -f deploy/crds/infinispan.org_infinispans_crd.yaml -n ${NAMESPACE}
 kubectl apply -f deploy/crds/infinispan.org_caches_crd.yaml -n ${NAMESPACE}
+kubectl apply -f deploy/crds/infinispan.org_backups_crd.yaml -n ${NAMESPACE}
```

Comment:
We should cleanup -n ${NAMESPACE} as CRD is not namespace'd object
```suggestion
kubectl apply -f deploy/crds/infinispan.org_backups_crd.yaml
```